conf.d/splunk.yaml

```diff
@@ -0,0 +1,8 @@
+---
+init_config:
+  default_timeout: 30
+
+instances:
+  - url: https://localhost:8089
+    username: obsrobot
+    password: foobar
```

Comment:
I fixed this by adding a new check for index copy healthy. Now we get index searchability and index copy health as separate checks.

In practice I suspect we'll modify this check more because we'll be more specific about the questions we wanted answered by monitoring. In some ways this version is a strawman.